spec/views/search_terms/index.html.tailwindcss_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe "search_terms/index", type: :view do
+  before(:each) do
+    assign(:search_terms, [
+      SearchTerm.create!(
+        keyword: "Keyword"
+      ),
+      SearchTerm.create!(
+        keyword: "Keyword"
+      )
+    ])
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/c919a05a12101ae07b4c21bf82ccf33697056e9e/spec/views/search_terms/index.html.tailwindcss_spec.rb#L12"></span>Indent the right bracket the same as the first position after the preceding left parenthesis.</td>
    </tr>
  </tbody>
</table>